### DISCUSSION TITLE
how to modify the format of alert message?

### DISCUSSION THREAD

**Discussion by ghost** [prometheus-operator/kube-prometheus#1246] [Q&A]
I want to modify the format the alert message, but didn't find the place to configure it from web ui, anyone can advise?

**Comment by paulfantom** [comment]
This is done by alertmanager receivers, take a look at https://github.com/thaum-xyz/ankhmorpork/blob/master/apps/monitoring/manifests/alertmanager/secret.yaml#L23-L76
